<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientLogin</class>
 <widget class="QDialog" name="ClientLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#8AD8FB;</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-family: sans-serif;
    font-size: 20px;
    color: #fff;
    background-color: #6188FF;
    padding: 12px 24px;
    border: 2px solid #063263; /* Adăugați contur cu grosimea și culoarea specificată */
    border-radius: 30px;
    cursor: pointer;
    transition: background-color 0.3s ease;
}

QPushButton:hover {
    background-color: #0056b3;
}
/* Stil pentru butoanele QPushButton */


</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>30</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sitka</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FFFFFF; /* Schimbăm culoarea textului în #063263 */
}</string>
   </property>
   <property name="text">
    <string>Client Login</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>240</width>
     <height>361</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0" colspan="3">
     <widget class="QLineEdit" name="Tara">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLineEdit" name="Judet">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLineEdit" name="Oras">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="3">
     <widget class="QLineEdit" name="CodPostal">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QLineEdit" name="Strada">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid #ccc;  /* Adaugă un contur de 2px solid cu o culoare gri deschis */
    border-radius: 5px;      /* Rotunjeste marginile */
    padding: 5px;            /* Adaugă un spațiu de 5px în interiorul lineEdit-ului */
    font-family: Arial, sans-serif; /* Setează fontul textului */
    font-size: 14px;         /* Setează dimensiunea fontului */
    color: #333;             /* Setează culoarea textului la un gri mai închis */
}

QLineEdit:hover {
    border-color: #aaa;      /* Schimbă culoarea conturului la un gri mai deschis atunci când mouse-ul este deasupra */
}

QLineEdit:focus {
    border-color: #0078d7;  /* Schimbă culoarea conturului la albastru atunci când lineEdit-ul are focus */
}</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="Numarul">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>34</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="Scara">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>34</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLineEdit" name="NumarApartament">
      <property name="minimumSize">
       <size>
        <width>60</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>34</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>520</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    font-family: sans-serif;
    font-size: 20px;
    color: #fff;
    background-color: #6188FF;
    padding: 12px 24px;
    border: 2px solid #063263; /* Adăugați contur cu grosimea și culoarea specificată */
    border-radius: 30px;
    cursor: pointer;
    transition: background-color 0.3s ease;
}

QPushButton:hover {
    background-color: #0056b3;
}
/* Stil pentru butoanele QPushButton */


</string>
   </property>
   <property name="text">
    <string>LogIn</string>
   </property>
  </widget>
  <widget class="QLabel" name="Header_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Account Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="Header_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Personal Data</string>
   </property>
  </widget>
  <widget class="QLabel" name="Header_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Adress Infos</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLineEdit" name="Email">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid #ccc;  /* Adaugă un contur de 2px solid cu o culoare gri deschis */
    border-radius: 5px;      /* Rotunjeste marginile */
    padding: 5px;            /* Adaugă un spațiu de 5px în interiorul lineEdit-ului */
    font-family: Arial, sans-serif; /* Setează fontul textului */
    font-size: 14px;         /* Setează dimensiunea fontului */
    color: #333;             /* Setează culoarea textului la un gri mai închis */
}

QLineEdit:hover {
    border-color: #aaa;      /* Schimbă culoarea conturului la un gri mai deschis atunci când mouse-ul este deasupra */
}

QLineEdit:focus {
    border-color: #0078d7;  /* Schimbă culoarea conturului la albastru atunci când lineEdit-ul are focus */
}</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="Parola">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>380</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid #ccc;  /* Adaugă un contur de 2px solid cu o culoare gri deschis */
    border-radius: 5px;      /* Rotunjeste marginile */
    padding: 5px;            /* Adaugă un spațiu de 5px în interiorul lineEdit-ului */
    font-family: Arial, sans-serif; /* Setează fontul textului */
    font-size: 14px;         /* Setează dimensiunea fontului */
    color: #333;             /* Setează culoarea textului la un gri mai închis */
}

QLineEdit:hover {
    border-color: #aaa;      /* Schimbă culoarea conturului la un gri mai deschis atunci când mouse-ul este deasupra */
}

QLineEdit:focus {
    border-color: #0078d7;  /* Schimbă culoarea conturului la albastru atunci când lineEdit-ul are focus */
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>202</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="verticalSpacing">
     <number>20</number>
    </property>
    <item row="0" column="0">
     <widget class="QLineEdit" name="Nume">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="Prenume">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="Telefon">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="DataNasterii">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>60</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
